package.json

```diff
@@ -34,7 +34,7 @@
     },
     "husky": {
         "hooks": {
-            "pre-commit": "npm run test && npm run lint:ci && npm run lint:styles"
+            "pre-commit": "npm run tsc && npm run test && npm run lint:ci && npm run lint:styles"
```

Comment:
It seems time to use here `npm-run-all`. The code will be mode shorter and readable.